applications/HDF5Application/python_scripts/utils.py

```diff
@@ -16,7 +16,7 @@
 
 
 def IsDistributed():
-    return KratosMultiphysics.DataCommunicator.GetDefault().IsDistributed()
+    return KratosMultiphysics.IsDitributedRun()
```

Comment:
this should also get a ModelPart and querry the DataComm
Or maybe it is not even needed to have this thin wrapper as the ModelPart also has the `IsDistributed` function

Reply:
I took a look, this fct is nly used for `CreateCoreSettings`
=> one could pass the `Model` there too and get the ModelPart from it for querriying `IsDistributed`

Reply:
I added the `model` as an extra parameters in `CreateCoreSettings`